chat-widget/src/components/livechatwidget/common/endChat.ts

```diff
@@ -132,31 +133,33 @@ const endChat = async (props: ILiveChatWidgetProps, chatSDK: any, state: ILiveCh
         }
     }
 
-    if (postMessageToOtherTab && !isNullOrEmptyString(uwid)) {
+    if (postMessageToOtherTab) {
         const endChatEventName = await getEndChatEventName(chatSDK, props);
         BroadcastService.postMessage({
             eventName: endChatEventName,
-            payload: uwid
+            payload: {
+                runtimeId: TelemetryManager.InternalTelemetryData.lcwRuntimeId
+            }
         });
     }
```

Comment:
Litlle confused, the issue is with start chat broadcasting the message and other opened tabs is responding to it creating new conversations, correct ? But the change is in endChat flow. It looks opposite ? Should not this supposed to be in startChat flow ? May be I am missing something. Would be great if you canhelp explain this one.

Reply:
Similar changes are applied to both the startChat and EndChat flow (the change to identify if the request originates from my own tab). If you look at the if statements of these2 closely, for startChat, we prevent other tabs to also startChat, and for endChat, we make sure all tabs close the conversation and reset context